envs/gridss/gridss-2.12.0.yaml

```diff
@@ -0,0 +1,341 @@
+name: gridss-2.11.1
```

Comment:
git doesn't allow to comment on the binary files, but above this yaml is a `TCRBOA7-T-WGS.input.png` and I am not sure it is part of the PR - just checking with you whether it should be there